cusub_common/drivers/gazebo_drivers/config/fakeyolo.yaml

```diff
@@ -4,9 +4,11 @@ yolo_faker:
  bounding_boxes_topic: 'cusub_perception/darknet_ros/bounding_boxes'
  sub_pose_topic: 'description/pose_gt'
  show_points: False
- global_box_noise: 0.2 # Multiplies bounds of box by width/height of box times 1/2 this
+#  global_box_noise: 0.2 # Multiplies bounds of box by width/height of box times 1/2 this
+ global_box_noise: 0.0 # Multiplies bounds of box by width/height of box times 1/2 this
```

Comment:
No noise, for shame...

Reply:
We need to average over the past few boxes to filter out noise, but a rolling average will lag the change in the box size as we get closer, so we need a fairly fast publisher of bounding boxes in order to do Jiangshi with noise, @skhadem optical flow.